prov/verbs/src/fi_verbs.h

```diff
@@ -528,7 +529,7 @@ static inline ssize_t fi_ibv_handle_post(ssize_t ret)
 						  -errno;
 			break;
 		default:
-			ret = -ret;
+			ret = -abs(ret);
```

Comment:
Clang is complaining about this in Travis CI. Do we need this since we already handle -ENOMEM?

Reply:
I changed type of passed value from `ssize_t` to `int`. IB Verbs functions return `int`.

> Do we need this since we already handle -ENOMEM?

We need this to convert other errors from negative to negative and also handle the behavior of mlx4/mlx5/hfi1 Verbs providers that returns positive value (i.e. convert from positive to negative).